src/main.dart

```diff
@@ -1,8 +1,105 @@
-// TODO: Run task list app
+import 'dart:io';
+
 void main(List<String> args) {
-  print('Hello World');
+  List<TaskListApp> tasks = [];
+  int taskId = 0;
+
+  while (true) {
+    print('What would you like to do?');
+    print('1. List all tasks');
+    print('2. Add a task');
+    print('3. Update a task');
+    print('4. Remove a task');
+    print('5. Exit');
+
+    String? choice = stdin.readLineSync();
+
+    switch (choice) {
+      case '1':
+        // List all tasks
+        for (TaskListApp task in tasks) {
+          print(task);
+        }
+
+        break;
+      case '2':
+        // Add a task
+        print('Enter task name:');
+        String name = stdin.readLineSync()!;
+        print('Enter task description:');
+        String description = stdin.readLineSync()!;
+        tasks.add(
+            TaskListApp(id: ++taskId, name: name, description: description));
```

Comment:
Before adding a task, null check. If i press enter and its empty it shouldnt let me continue